lib/ruby_lsp/requests/definition.rb

```diff
@@ -78,8 +84,8 @@ def initialize(document, global_state, position, dispatcher, sorbet_level)
             parent,
             position,
           )
-        elsif target.is_a?(Prism::CallNode) && target.name != :require && target.name != :require_relative &&
-            !covers_position?(target.message_loc, position)
+        elsif target.is_a?(Prism::CallNode) && !covers_position?(target.message_loc, position) &&
+            !SPECIAL_METHOD_CALLS.include?(target.message)
```

Comment:
Wouldn't it be more performant to check first that the call node is not one of the special method calls?
```suggestion
        elsif target.is_a?(Prism::CallNode) && !SPECIAL_METHOD_CALLS.include?(target.message) && !covers_position?(target.message_loc, position)
```
which was what the old code was doing.